src/collection.js

```diff
@@ -209,6 +209,29 @@ export default class Collection {
 		return item || null;
 	}
 
+	/**
+	 * Returns a boolean indicating whether the collection contains an item with the specified id or index.
+	 *
+	 * @param {String|Number} idOrIndex The item id or index in the collection.
+	 * @returns {Boolean} `true` if the collection contains the item, `false` otherwise.
+	 */
+	has( idOrIndex ) {
+		let item;
+
+		if ( typeof idOrIndex == 'string' ) {
+			return this._itemMap.has( idOrIndex );
+		} else if ( typeof idOrIndex == 'number' ) {
```

Comment:
I'm not sure if checking index is a way to go. I'd expect something similar to `[].includes()` with additions for our needs.